src/patternfly/components/CustomSearchInput/examples/CustomSearchInput.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Custom search input'
+beta: true
+section: components
+cssPrefix: pf-c-custom-search-input
+---
+
+## Examples
+```hbs title=Basic
+{{#> custom-search-input custom-search-input--placeholder="Find by name"}}
+{{/custom-search-input}}
+```
+
+```hbs title=No-match
+{{#> custom-search-input custom-search-input--placeholder="Find by name" custom-search-input--value="Joh"}}
+{{/custom-search-input}}
+```
+
+```hbs title=Match-with-result-count
+{{#> custom-search-input custom-search-input--placeholder="Find by name" custom-search-input--value="John Doe" custom-search-input--count="3"}}
+{{/custom-search-input}}
+```
+
+```hbs title=Match-with-navigable-options
+{{#> custom-search-input custom-search-input--placeholder="Find by name" custom-search-input--value="John Doe" custom-search-input--count="1/3" custom-search-input--IsNavigable="true" custom-search-input--IsFirstMatch="true"}}
+{{/custom-search-input}}
+```
+
+### Accessibility
+| Attributes | Applied to | Outcome |
+| -- | -- | -- |
+| `aria-hidden="true"` | `.pf-c-custom-search-input__icon > *` | Hides the search icon from assistive technologies. **Required** |
+| `aria-label="Previous"` | `.pf-c-custom-search-input__nav > .pf-c-button` | Provides an accessible label for the previous nav button. **Required** |
+| `aria-label="Next"` | `.pf-c-custom-search-input__nav > .pf-c-button` | Provides an accessible label for the next nav button. **Required** |
+| `aria-label="[descriptive text]"` | `.pf-c-custom-search-input__text-input` | Provides an accessible label for the search input. **Required** |
+| `aria-label="Clear"` | `.pf-c-custom-search-input__close > .pf-c-button` | Provides an accessible label for the close button. **Required** |
+
+### Usage
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-custom-search-input` | `<div>` | Initiates the custom search input component. **Required** |
+| `.pf-c-custom-search-input__text` | `<span>` | Initiates the text area. **Required** |
+| `.pf-c-custom-search-input__text-input` | `<input>` | Initiates the search input. **Required** |
```

Comment:
Do you need `.pf-c-custom-search-input__text` and `.pf-c-custom-search-input__text-input`?

Reply:
Yeah, it's to manage the 2px blue bottom border when focused on the text input and to turn the magnifying icon dark when hovered/focused on the text input.